src/dotnet/Chat.Service/Db/DbChat.cs

```diff
@@ -7,7 +7,7 @@ namespace ActualChat.Chat.Db;
 
 [Table("Chats")]
 [Index(nameof(CreatedAt))]
-[Index(nameof(Version), nameof(Id))]
+[Index(nameof(Version), nameof(Id), nameof(IsPlaceRootChat))]
```

Comment:
Can't strictly prove it, but I have an impression that we need only index by Version field with included Id and IsPlaceRootChat. I think inculding Id as second prop in the index does not much improve ordering by Id.
Indexes with included fields can be defined the following way https://learn.microsoft.com/en-us/ef/core/modeling/indexes?tabs=data-annotations#included-columns
It's discussable which kind of index we should use to improve ListChanged query performance.

Reply:
done, please check